client/src/app/pages/applications/components/application-detail-drawer/fact-detail-modal/fact-code-snip-viewer.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { CodeEditor, Language } from "@patternfly/react-code-editor";
+import { Fact } from "@app/api/models";
+
+export interface IFactCodeSnipViewerProps {
+  fact: Fact;
+}
+
+export const FactCodeSnipViewer: React.FC<IFactCodeSnipViewerProps> = ({
+  fact,
+}) => {
+  return (
+    <CodeEditor
+      isReadOnly
+      isDarkTheme
+      isLineNumbersVisible
+      language={Language.json}
+      height="450px"
+      code={JSON.stringify(fact.data)}
+    />
```

Comment:
It took a little bit of configuring in https://github.com/konveyor/tackle2-ui/pull/966/files, but not much! now we just have it for wherever we want it

Reply:
Now it would be really good to get code splitting working so the main app bundle isn't too huge.

PS - I know how to get it working, just haven't worked it up into a skinny PR yet.

Reply:
We would love the help with code splitting! It's something I've never really taken the time to figure out.